tools/mmp/driver.cs

```diff
@@ -348,6 +349,7 @@ static void Main2 (string [] args)
 				{ "http-message-handler=", "Specify the default HTTP Message Handler", v => { http_message_provider = v; }},
 				{ "extension", "Specifies an app extension", v => is_extension = true },
 				{ "allow-unsafe-gac-resolution", "Allow MSBuild to resolve from the System GAC", v => {} , true }, // Used in Xamarin.Mac.XM45.targets and must be ignored here. Hidden since it is a total hack. If you can use it, you don't need support
+				{ "allow-unsafe-full-linking", "Allow linking when targetting XM Full Profile. This is very unsafe unless you know exactly you are doing.", v => force_all_full_linking = true , true }, // Undocumented for a reason, You get to keep the pieces when it breaks
```

Comment:
Bad name and description. Full is unclear between full-vs-sdk linking and full profile.
Use something like `force-unsupported-linker` and let's not say it's unsafe in the description when it's unsupported (not the same).